articles/multifactor-authentication/api/manage.md

```diff
@@ -1,16 +1,34 @@
 ---
 title: Manage the Authenticators for Multifactor Authentication
 description: How to manage your MFA authenticators
-beta: true
 ---
 
 # Manage the Authenticators
 
-::: warning
-This article includes documentation on features that are still under development. These features are available to customers with early access.
-:::
+Auth0 provides several API endpoints to help you manage the authenticators you're using with an application for multifactor authentication (MFA).
 
-Auth0 provides several API endpoints to help you manage the authenticators you're using with a particular tenant for multifactor authentication (MFA).
+## Before you start
+
+The MFA endpoints require an [Access Token](/tokens/access-token) with:
+
+- `audience`: Set to `https://${account.namespace}/mfa`
+- `scope`: Include `enroll` for enrollment, `read:authenticators` to list authenticators, and `remove:authenticators` to delete authenticators.
```

Comment:
This is **really important information** and I don't remember reading it in the other PR, for the Authentication API. People must know how to get a valid token for each endpoint.